src/monitors/BranchMonitor.v3

```diff
@@ -4,30 +4,22 @@
 def monitor_ = MonitorRegistry.add(
 	"branches", "Tallies direct and indirect branches.",
 	BranchMonitor.new());
-def inCodeMode_ = monitor_ // TODO: return Option<int>
-	.addOption("code|c", "emit report with code");
-def showZeroCounts_ = monitor_ // TODO: return Option<bool>
-	.addOption("zero|z", "show zero counts");
+def inCodeModeOpt_ = monitor_.option_group.addAlias(
```

Comment:
I think it would be better to avoid the extra lines of having a set method and instead just use the `Option` object directly at use sites. If it's just because there are two aliases for a command, maybe it'd be better to add an alias mechanism that accepts an `Option` object, so there is only one storage location for the state.